test/components/previews/nested_form_input_component_preview.rb

```diff
@@ -0,0 +1,24 @@
+class NestedFormInputComponentPreview < ViewComponent::Preview
+
+  include ActionView::Helpers::TagHelper
+  include ActionView::Helpers::FormTagHelper
+
+  # @param object_name text
+  def default(object_name: 'contact')
+    render NestedFormInputsComponent.new(object_name: object_name) do |c|
+      c.header do
+        content_tag(:div, 'Contact name', class: 'w-50 mx-1') + content_tag(:div, 'Contact email', class: 'w-50 mx-1')
+      end
+
+      c.template do
```

Comment:
Replace this template with the template used in upload ontology page #292

Reply:
resolve this in this PR https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/292